src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -28,6 +44,23 @@ public static ArgumentMultimap tokenize(String argsString, Prefix... prefixes) {
         return extractArguments(argsString, positions);
     }
 
+    /**
+     * @author Lookaz
+     * Checks a given argument string and returns the corresponding
+     * CommandMode enum
+     * @param args argument string to check
+     * @return CommandMode enum type if available, else CommandMode.INVALID
+     */
+    public static CommandMode checkMode(String args) {
+        String trimmedArgs = args.trim();
+        if (!Character.toString(trimmedArgs.charAt(MODE_POSITION + 1)).equals(" ")) {
```

Comment:
What does `trimmedArgs.charAt(MODE_POSITION + 1)` here do?

Reply:
trimmArgs represent the argument string but without the leading and trailing whitespaces. trimmedArgs.charAt(MODE_POSITION + 1) checks if the character after the mode number is a white space.